### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by BTWS2**
This pull request fixes typos using RubyMine and the [Grazie](https://plugins.jetbrains.com/plugin/12175-grazie) plugin.

### CODE REVIEW

File: config/locales/views/defaults/en.yml [hunk 1]
```diff
@@ -38,7 +38,6 @@ en:
       my_exports: My exports
       submissions: Submissions
       users: Users
-      courses: Courses
```
Comment (thepieterdc) [inline config/locales/views/defaults/en.yml:41]: Why was this removed?

File: config/locales/views/defaults/nl.yml [hunk 2]
```diff
@@ -38,7 +38,6 @@ nl:
       my_exports: Mijn exports
       submissions: Oplossingen
       users: Gebruikers
-      courses: Cursussen
```
Comment (thepieterdc) [inline config/locales/views/defaults/nl.yml:41]: Why was this removed?

File: config/locales/views/defaults/en.yml (hunk 1 above)
Comment (BTWS2) [reply config/locales/views/defaults/en.yml:41]: It was defined twice
image

File: config/locales/views/defaults/nl.yml (hunk 2 above)
Comment (BTWS2) [reply config/locales/views/defaults/nl.yml:41]: It was defined twice
image

**Review by chvp — APPROVED** [review]

**Review by bmesuere — APPROVED** [review]
